osu.Game.Rulesets.Osu.Tests/TestSceneLegacyBeatmapSkin.cs

```diff
@@ -24,34 +26,81 @@ public class TestSceneLegacyBeatmapSkin : ScreenTestScene
         [Resolved]
         private AudioManager audio { get; set; }
 
-        [TestCase(true)]
-        [TestCase(false)]
-        public void TestBeatmapComboColours(bool customSkinColoursPresent)
+        private readonly Bindable<bool> beatmapSkins = new Bindable<bool>();
+        private readonly Bindable<bool> beatmapColours = new Bindable<bool>();
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            config.BindWith(OsuSetting.BeatmapSkins, beatmapSkins);
+            config.BindWith(OsuSetting.BeatmapColours, beatmapColours);
+        }
+
+        [TestCase(true, true, true)]
+        [TestCase(true, false, true)]
+        [TestCase(false, true, true)]
+        [TestCase(false, false, true)]
+        public void TestBeatmapComboColours(bool userHasCustomColours, bool useBeatmapSkin, bool useBeatmapColour)
```

Comment:
Not a huge deal but pretty weird to have `useBeatmapColour` as a param that is always true in a test called `TestBeatmapComboColours`. Would just inline the `true`, probably. Same applies to the other tests.

Reply:
I had the test groups stacked together originally and split them out into separate groups to make things a bit clearer. I forgot to remove the extra param since it should be inlined like you mentioned.